lab_1_keywords_tfidf/main.py

```diff
@@ -20,9 +25,16 @@ def check_list(user_input: Any, elements_type: type, can_be_empty: bool) -> bool
     Returns:
         bool: True if valid, False otherwise
     """
+    if not isinstance(user_input, list):
+        return False
+    if not can_be_empty and len(user_input) == 0:
+        return False
+    return all(isinstance(element, elements_type) for element in user_input)
+
 
 
-def check_dict(user_input: Any, key_type: type, value_type: type, can_be_empty: bool) -> bool:
+def check_dict(user_input: Any, key_type: type, value_type:
```

Comment:
.

Reply:
In laboratory works it is prohibited to change docstrings and defs of functions. Current limit about length of strings is 100 characters. I really appreciate that you want to make our code better, it is really cool, but our code is aligned with PEP